lang/performous.pot

```diff
@@ -1,2205 +1,2205 @@
+#. value: false
+#: ../data/config/schema.xml:389
+msgid "Suppress center channel"
+msgstr ""
+
+#. Suppress center channel
+#. Suppress audio of center channel (e.g. vocals).
+#. name: audio/suppress_center_channel
+#. type: bool
+#. value: false
+#: ../data/config/schema.xml:390
+msgid "Suppress audio of center channel (e.g. vocals)."
+msgstr ""
+
+#. Song folders
+#. Where to recursively look for songs. DATADIR at the beginning means all Performous data folders.
+#. name: paths/songs
+#. type: string_list
+#. hidden: false
+#: ../data/config/schema.xml:395
+msgid "Song folders"
+msgstr ""
+
+#. Default Song folders
+#. Where to recursively look for songs. DATADIR at the beginning means all Performous data folders.
+#. name: paths/system-songs
+#. type: string_list
+#: ../data/config/schema.xml:418
+msgid ""
+"Where to recursively look for songs. DATADIR at the beginning means all "
+"Performous data folders."
+msgstr ""
+
+#. Show hidden folders
+#. When true the screen 'Paths' will also show hidden folders.
+#. name: paths/showhiddenfolders
+#. type: bool
+#. value: false
+#: ../data/config/schema.xml:400
+msgid "When true the screen 'Paths' will also show hidden folders."
+msgstr ""
+
+#. Default Song folders
+#. Where to recursively look for songs. DATADIR at the beginning means all Performous data folders.
+#. name: paths/system-songs
+#. type: string_list
+#: ../data/config/schema.xml:417
+msgid "Default Song folders"
+msgstr ""
+
+#. Base folders for data
+#. System defaults are included automatically. Additional paths can be added here.
+#. name: paths/system
+#. type: string_list
+#: ../data/config/schema.xml:421
+msgid "Base folders for data"
+msgstr ""
+
+#. Base folders for data
+#. System defaults are included automatically. Additional paths can be added here.
+#. name: paths/system
+#. type: string_list
+#: ../data/config/schema.xml:422
+msgid ""
+"System defaults are included automatically. Additional paths can be added "
+"here."
+msgstr ""
+
+#. Sort order
+#. Currently active sort order.
+#. name: songs/sort-order
+#. type: int
+#. value: 0
+#. hidden: true
+#: ../data/config/schema.xml:429
+msgid "Currently active sort order."
+msgstr ""
+
+#: ../data/performous.desktop:3
+msgid "Performous"
+msgstr ""
+
+#: ../data/performous.desktop:4
+msgid "A karaoke game"
+msgstr ""
+
+#: ../data/performous.desktop:17
+msgid "song;music;dance;karaoke;microphone;sound;singing;"
+msgstr ""
+
+#: ../game/i18n.hh:63 ../game/i18n.hh:100
+msgid "Auto"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Guitar"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Coop guitar"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Rhythm guitar"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Bass"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Drums"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Vocals"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Harmonic 1"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Harmonic 2"
+msgstr ""
+
+#: ../game/song.hh:28
+msgid "Harmonic 3"
+msgstr ""
```

Comment:
It might be that i'm just overlooking things, but this PR should fix that enumerations within schema.xml gets translated, but i don't see `../data/config/schema.xml:133` (https://github.com/performous/performous/blob/master/data/config/schema.xml#L132-L137) within the main `performous.pot` ? I do see them in the `de.po`

Reply:
Fixed